src/extensions/animation/controls.js

```diff
@@ -24,15 +22,17 @@ import {
 	MenuGroup,
 	Popover,
 	Toolbar,
-	Tip
+	Tip,
 } from '@wordpress/components';
 import { check } from '@wordpress/icons';
 import { withSelect } from '@wordpress/data';
 
 function PreviewAnimationPopover( { hoveredAnimation, selected } ) {
-	if ( ! hoveredAnimation ) return null;
+	if ( ! hoveredAnimation ) {
+		return null;
+	}
 
-	const block = _.cloneDeep(selected);
+	const block = _.cloneDeep( selected );
```

Comment:
Are you using a linter for that? I'm sorry for all of the lint fixes 😓

Reply:
Yeah, it's all automated in my IDE. I just run the formatter and commit the changes :)